src/sbt-test/sbt-1.5/scala-3/build.sbt

```diff
@@ -0,0 +1,19 @@
+ThisBuild / scalafixDependencies += "ch.epfl.scala" %% "example-scalafix-rule" % "1.6.0"
```

Comment:
This contains very simple syntactic and semantic rules so it's good for an initial scripted until built-in rules pick up actual support. The only downside is that the semantic rule does not even evaluate the semantic document, so I am not sure it really tests the generation of semanticdb compiler output... Maybe we could tag a 1.6.1 with https://github.com/scalacenter/sbt-scalafix/pull/125/files#diff-38f3b339536ccc14af957f820b7bfda9be5e1226210bb2d8938d7fab17003911R7 to lift this doubt?